### PR TITLE
Added a comic state widget to the home page [#1224]

### LINKED ISSUE #1224: The home page shows the comic state distribution
As an admin, I'd like to see a distribution of comics in my library across all of the defined states.

### PR DESCRIPTION
**Pull request by mcpierce**
Moved the library statistics display to its own component,
and added a new component for the comic state distribution.

Changed the home page layout to use a tile system.

Also fixed a bug that kept teams from being shown.

Closes #1224 

## Status
READY

## Does this PR contain migrations?
NO

## Before You Submit Your PR:
- [ ] Have you announced your PR on the comixed-dev mailing list?
- [X] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Refactors existing code (code changes for efficiency or maintainability)
- [ ] Security fix (be sure to include the CVE in the commit message) 
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [X] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

File: comixed-webui/src/assets/i18n/pt/app.json [hunk 1]
```diff
@@ -56,21 +56,26 @@
...
+    "legend": {
+      "collections-top-entries-x-axis": "Number Of Comics",
+      "comic-state-x-axis": "Number Of Comics In Each State"
+    },
     "tab-title": "Início",
     "title": {
+      "comic-collections-chart": "Top Collection Entries",
+      "comic-state-chart": "Comics By State",
       "comic-statistics": "Estatísticas da Biblioteca de Comics",
       "loading-comics": "Carregando Biblioteca..."
```
Comment (jmcrfp) [inline comixed-webui/src/assets/i18n/pt/app.json:80]: If you want you can add the portuguese translation also

"comic-state-added": "Adicionados",
"comic-state-changed": "Alterados",
"comic-state-deleted": "Apagados",
"comic-state-stable": "Estáveis",
"comic-state-unprocessed": "Por Processar",
"select-collection-type": "Tipo De Coleção",

"collections-top-entries-x-axis": "Quantidade De Comics",
"comic-state-x-axis": "Quantidade De Comics Por Estado"

"comic-collections-chart": "Principais Entradas Da Coleção",
"comic-state-chart": "Comics Por Estado",

**Comment by jmcrfp** [comment]
Everything looks nice, but when I was looking I found something that is not so great but it was probably already there:
So the right box (Comics by state) doesn't show the right border. 
And that's probably because the browser vertical scroll bar appears.
And the reason the it appears is because the default menu side bar vertical size is bigger than the page it self. (I know the menu is bigger that normal page size and it has it own vertical scroll bar, and that's fine) But now I need to scroll 2 bars to see the bottom of the menu.

File: comixed-webui/src/assets/i18n/pt/app.json (hunk 1 above)
Comment (mcpierce) [reply comixed-webui/src/assets/i18n/pt/app.json:80]: Thanks. Though I'm going to leave translations until the 1.1 release. That way we keep the translations all together.

**Comment by mcpierce** [comment]
> Everything looks nice, but when I was looking I found something that is not so great but it was probably already there: So the right box (Comics by state) doesn't show the right border. And that's probably because the browser vertical scroll bar appears. And the reason the it appears is because the default menu side bar vertical size is bigger than the page it self. (I know the menu is bigger that normal page size and it has it own vertical scroll bar, and that's fine) But now I need to scroll 2 bars to see the bottom of the menu.

I think you're right about it likely already being there. We should probably do some tweaks in a separate commit to clean up the layout. Right now the main application container is set to 100% width. I'll start up a thread in the dev list for how to properly address that.

**Pull request merged**